openfisca_us/variables/demographic/person.py

```diff
@@ -43,3 +43,57 @@ def formula(person, period, parameters):
             [age < 18, age < 65, age >= 65],
             [AgeGroup.CHILD, AgeGroup.WORKING_AGE, AgeGroup.SENIOR],
         )
+
+
+class ccdf_age(Variable):
+    value_type = float
+    entity = Person
+    label = u"CCDF age"
+    definition_period = YEAR
+
+
+class CCDFCareLocation(Enum):
+    CENTER_BASED = "Center-based"
+    HOME_BASED = "Home-based"
+
```

Comment:
@MaxGhenis @nmrodelo feel free to override or if we should make this a general pattern, but I wonder if for all enums we should have an "UNSPECIFIED"/"UNKNOWN" value?

Reply:
What logic would we use in that case though?

Reply:
True, maybe it's better not to- it'd need an UNKNOWN path all the way through the computation tree.